src/restic/fs/file.go

```diff
@@ -102,6 +102,12 @@ func Symlink(oldname, newname string) error {
 	return os.Symlink(fixpath(oldname), fixpath(newname))
 }
 
+// Hardlink creates newname as a hard link to oldname.
+// If there is an error, it will be of type *LinkError.
+func Hardlink(oldname, newname string) error {
```

Comment:
Why is this function called `Hardlink()`, an not just `Link()` as the corresponding function in package `os`?